src/templates/shared/bits.js

```diff
@@ -1,27 +1,40 @@
 const commonTags = require("common-tags");
 
+const DISCORD_URL = "https://discord.reclaimers.net";
+const REPO_URL = "https://github.com/Sigmmma/c20";
+
+//converts a title into a URL- or ID-friendly slug
+const slugify = (title) => title
+  .toLowerCase()
+  .replace(/['-]/g, "")
+  .replace(/[^a-z0-9]/g, " ")
+  .split(" ")
+  .filter(part => part.length > 0)
+  .join("-");
+
+const escapeHtml = (s) => commonTags.safeHtml`${s}`;
 const html = commonTags.stripIndent(commonTags.html);
 
 const anchor = (href, body) => html`
   <a href="${href}">${body}</a>
 `;
 
-const pageAnchor = (page) => anchor(page._dirUrl, page.title);
+const pageAnchor = (page) => anchor(page._dirUrl, escapeHtml(page.title));
```

Comment:
I started sprinkling some `escapeHtml` around places where we directly inject metadata or headers into the markup. For example, the headers on the bitmaps page which contain tool.exe errors were causing some rendering issues because of their "<" characters.